packages/core/documentation/src/components/DocumentationContent.vue

```diff
@@ -2,11 +2,51 @@
   <div class="documentation">
     <div v-if="documentList && !documentList.length">
       <KCard v-if="emptyStateCard">
+        <EntityEmptyState
+          v-if="enableV2EmptyStates"
+          :action-button-text="t('documentation.show.empty_state_v2.cta')"
+          appearance="secondary"
+          :can-create="() => canEdit()"
```

Comment:
question: Is it really `canEdit()`? That doesn't seem right.

Reply:
the previous actions buttons used the same canEdit prop. 
<img width="678" alt="Screenshot 2025-02-19 at 12 18 31 AM" src="https://github.com/user-attachments/assets/af13dc69-25fe-470b-9a06-4f9f4c3ee00e" />

Reply:
This is the part I needed to see:

<img width="680" alt="image" src="https://github.com/user-attachments/assets/7126dc5c-df0a-479b-9c82-2afefe865b30" />